api/views/healthCheck/healthCheck.py

```diff
@@ -403,6 +404,147 @@ def health_check_yang_search_admin():
         return make_response(jsonify(error_response(service_name, err)), 200)
 
 
+@app.route('/yangcatalog', methods=['GET'])
+def health_check_yangcatalog():
+    service_name = 'yangcatalog'
+    status = 'running'
+    message = 'All URLs responded with status code 200'
+    additional_info = []
+
+    urls = ['http://yangcatalog.org',
+            'http://www.yangcatalog.org',
+            'https://yangcatalog.org',
+            'https://www.yangcatalog.org',
+            'http://yangvalidator.com',
+            'http://www.yangvalidator.com',
+            'https://yangvalidator.com',
+            'https://www.yangvalidator.com',
+            'http://18.224.127.129',
+            'https://18.224.127.129',
+            'http://[2600:1f16:ba:200:a10d:3212:e763:e720]',
+            'https://[2600:1f16:ba:200:a10d:3212:e763:e720]'
+            ]
+
+    for url in urls:
+        result = {}
+        result['label'] = url
+        try:
+            if any(char.isdigit() for char in url):
+                response = requests.get(url, verify=False)
+            else:
+                response = requests.get(url)
+            status_code = response.status_code
+            app.LOGGER.info('URl: {} Status code: {}'.format(url, status_code))
+            result['message'] = '{} OK'.format(status_code)
+        except:
+            result['message'] = '500 NOT OK'
+            status = 'problem'
+            message = 'Problem occured, see additional info'
+        additional_info.append(result)
+
+    return make_response(jsonify({'info': '{} is available'.format(service_name),
+                                  'status': status,
+                                  'message': message,
+                                  'additional_info': additional_info}), 200)
+
+
+@app.route('/confd-full-check', methods=['GET'])
+def health_check_confd_full_check():
+    service_name = 'ConfD - Full check'
+    status = 'running'
+    message = 'All requests responded with status code 200'
+    additional_info = []
+    check_module_name = 'confd-full-check'
+    confd_prefix = '{}://{}:{}'.format(yc_gc.protocol, yc_gc.confd_ip, repr(yc_gc.confdPort))
+    headers = {'Content-type': 'application/yang-data+json', 'Accept': 'application/yang-data+json'}
+
+    try:
+        # GET
+        result = {}
+        result['label'] = 'GET yang-catalog@2018-04-03'
```

Comment:
I don t like the idea of playing with something that already exists in our database and accidentally removing it. especially with latest yang-catalog file that is used with every search. Although we are using redis I still want this database to be clean. you can create some json file output and save it into project out of this file but change name revision, organization and namespace. make first PUT request and make sure it is in there and continue from there as you did here with yang-catalog.

Reply:
yang-catalog@2018-04-03 is essential module which need to be in ConfD in first place, this is the reason why I am making GET of this module ... then I change name, which is basically same as loading some fictional module from json and using it for next checks ... However it is not a problem to move this to some json file and loading from it.

Reply:
Oh sorry, I have missed that. You are right this is ok.